docs/userguide/tasks.rst

```diff
@@ -785,6 +785,15 @@ You can also set `autoretry_for`, `max_retries`, `retry_backoff`, `retry_backoff
     and the actual delay value will be a random number between zero and that
     maximum. By default, this option is set to ``True``.
 
+.. versionadded:: 5.2.4
```

Comment:
i think this is a new feaure and should be in 5.3.0 :)

Reply:
I will include this in my next push.